microprofile/galleon-common/src/main/resources/modules/system/layers/base/org/jboss/narayana/rts/lra-coordinator/main/module.xml

```diff
@@ -0,0 +1,53 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!--
+  ~ JBoss, Home of Professional Open Source.
+  ~ Copyright 2023, Red Hat, Inc., and individual contributors
+  ~ as indicated by the @author tags. See the copyright.txt file in the
+  ~ distribution for a full listing of individual contributors.
+  ~
+  ~ This is free software; you can redistribute it and/or modify it
+  ~ under the terms of the GNU Lesser General Public License as
+  ~ published by the Free Software Foundation; either version 2.1 of
+  ~ the License, or (at your option) any later version.
+  ~
+  ~ This software is distributed in the hope that it will be useful,
+  ~ but WITHOUT ANY WARRANTY; without even the implied warranty of
+  ~ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+  ~ Lesser General Public License for more details.
+  ~
+  ~ You should have received a copy of the GNU Lesser General Public
+  ~ License along with this software; if not, write to the Free
+  ~ Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+  ~ 02110-1301 USA, or see the FSF site: http://www.fsf.org.
+  -->
+
+<module xmlns="urn:jboss:module:1.9" name="org.jboss.narayana.rts.lra-coordinator">
+
+  <properties>
+    <property name="jboss.api" value="private"/>
+  </properties>
+  <resources>
+    <artifact name="${org.jboss.narayana.rts:lra-service-base}"/>
+    <artifact name="${org.jboss.narayana.rts:lra-coordinator-jar}"/>
+  </resources>
+
+  <dependencies>
+    <!-- lra-service-base -->
+    <module name="org.eclipse.microprofile.lra.api"/>
+    <module name="jakarta.ws.rs.api" />
+    <module name="org.jboss.logging"/>
+    <!-- lra-coordinator-jar -->
+    <module name="org.jboss.jts"/>
+    <module name="org.jboss.resteasy.resteasy-jaxrs"/>
+    <module name="jakarta.enterprise.api"/>
+    <module name="jakarta.servlet.api"/>
+    <module name="org.eclipse.microprofile.openapi.api"/>
+    <module name="io.smallrye.fault-tolerance" optional="true"/>
```

Comment:
Can you add a comment documenting why this is optional? That will help future maintainers.

Reply:
hmm, it shouldn't; coordinator jar uses FT.